### PR TITLE
Suggestion for H1 and Title FIX #111

### LINKED ISSUE #111: Suggestion for H1 and Title tags
Hi Breck (@breck7),

The home page of the PLDB site lacks some details that are actually spelled out very carefully elsewhere on the site.

For example, the abbreviation PLDB itself. You and I know what it means, but to the uninformed observer, especially someone coming to the main PLDB page, they might well ask: _"What does PLDB mean?"_

A glance down the page would show that it might be about programming languages, but it is not very evident from first looking at the page.

The **About** page does a better job of explaining the meaning and purpose of PLDB, but some of these details should be front-and-center on the main page. 

For the search engines, these key details could be put in the H1 and Title tags and would give search users a clearer understanding of what the site is about.  Also, for the best SEO impact, both the Title and H1 tag should be the same. 

The current title tag for the main PLDB page is:

pldb-title-tag

This is, to say the least, ahmm..... riveting! 😄 

The **Title** tag is what appears on the browser tab and is (usually) what first appears in the search engine results. **PLDB**, by itself, is not very informative.

The H1 tag is used by the search engines to categorize the content on the page. The recommendation is that there is only one H1 tag on a page and that it should give a concise summary of the page content.

The current H1 tag does not contain any text at all and is used only for linking:

h1-tag-no-text

I would consider this to be a missed opportunity.

Since it is recommended that the Title tag be 65 characters or fewer, my suggestion for the Title / H1 tags would be:

**PLDB - A Comprehensive Database of Programming Languages and Features**

Ok, it is 69 characters long, and the last few characters might be chopped off in the search results, but this might be something to consider.

I see that you already have a PLDB "Header" just before the first search form:

pldb-header

It may be difficult to incorporate these  H1 / Title tags changes, given the current design of the page, but I believe that it would be an important part of  giving out a clear message of what the PLDB is all about.

Again, this is just a suggestion. Feel free to ignore if it is too much work!

Kind Regards,
Liam

### PR DESCRIPTION
**Pull request by adriantintpilver**
Suggestion for H1 and Title FIX #111

### CODE REVIEW

**Comment by breck7** [comment]
It's better but I'm not crazy about it. @adriantintpilver can you list 5 options?

**Comment by adriantintpilver** [comment]
Hi!
I give you a list of words that I think we could tag in the order that for me would be from the most important to the least important.

Database Programming Languages Development Software Applications Techie Geek Systems Library IT-specialist Analyst Expert Codification

For google searches, it is not good to put more than 5 or 6 words since it is losing effectiveness.

what do you think?

Thanks!
Regards!

File: site/index.scroll [hunk 1]
```diff
@@ -1,4 +1,7 @@
+title PLDB - A Comprehensive Database of Programming Languages and Features
+
 import settings.scroll
+htmlTitle PLDB - A Comprehensive Database of Programming Languages and Features
```
Comment (breck7) [inline site/index.scroll:4]: What if we just kept it short:

```
htmlTitle PLDB - A Programming Language Database
```

Comment (adriantintpilver) [reply site/index.scroll:4]: it's ok for me, the short version, potence the words in there, in this case, this 3 words Programming Language Database, and think those 3 words are the ones that best reflect what the PLDB is.

File: site/index.scroll [hunk 2]
```diff
@@ -1,4 +1,7 @@
+title PLDB - A Comprehensive Database of Programming Languages and Features
```
[the remark this answers is not in the document]
Comment (adriantintpilver) [reply site/index.scroll:1]: yes the two tag, do the same function, keeping only htmltitle it's ok.

**Comment by adriantintpilver** [comment]
I sent a new pull request with this change to the short version.

**Pull request merged**